README.md

```diff
@@ -19,27 +19,27 @@ You can find CogniCrypt<sub>SAST</sub> also on [Maven Central](https://central.s
 
 CogniCrypt<sub>SAST</sub> uses Maven as build tool. You can compile and build this project via
 
-```mvn package -DskipTests=true```.
+```mvn clean package -DskipTests=true```.
 
-A packaged  `jar` artifact including all dependency is found in `CryptoAnalysis/build/CryptoAnalysis-x.y.z-jar-with-dependencies.jar`. Building requires at least Java 11.
+The packaged  `jar` artifacts including all dependencies can be found in `/apps`. Building requires at least Java 11.
 
 ## Usage
 
-CogniCrypt<sub>SAST</sub> can be started in headless mode (i.e., detached from Eclipse) via the class `crypto.HeadlessCryptoScanner`. It requires two arguments: 
+CogniCrypt<sub>SAST</sub> can be started in headless mode (i.e., detached from Eclipse) via the file `HeadlessCryptoScanner-x.y.z-jar-with-dependencies.jar`. It requires two arguments: 
 * The path to the directory of the CrySL (source code format) rule files. The source code for the rules which contain specification for the JCA is found [here](https://github.com/CROSSINGTUD/Crypto-API-Rules).
 * The path of the application to be analyzed (.jar file or the root compilation output folder which contains the .class files in subdirectories)
 
 ```
-java -cp <path-to-analysis-jar> crypto.HeadlessCryptoScanner 
+java -jar HeadlessCryptoScanner-3.2.1-jar-with-dependencies.jar 
```

Comment:
Why go with a specific version?

Do we need to update the README.md manually for each version or will the version handling take care?